src/main/java/solvd/laba/ermakovich/hf/event/account/CreateAccountResponseFactory.java

```diff
@@ -0,0 +1,29 @@
+package solvd.laba.ermakovich.hf.event.account;
+
+import java.time.LocalDateTime;
+import solvd.laba.ermakovich.hf.event.EventRoot;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+public class CreateAccountResponseFactory {
+
+    public EventRoot getFailedEvent(CreateAccount event) {
```

Comment:
you can simplify your method names

can be just `buildFailed` or `failed`
and `buildSuccess` or `success`